src/services/getPersonDates.ts

```diff
@@ -0,0 +1,34 @@
+import { useContext } from 'react';
+import { PeopleContext } from '../store/PeopleContent';
+
+function ExtractDates(name: string | null) {
+  const { people } = useContext(PeopleContext);
+  const parentOfKid = people.find(personEl => personEl.name === name);
+
+  if (!parentOfKid) {
+    return undefined;
+  }
+
+  const parentBirthDate = parentOfKid?.born;
+
+  const parentNameDivided = parentOfKid?.name
+    .split(' ')
+    .join('-')
+    .toLowerCase();
+
+  const parentDates = parentNameDivided + '-' + parentBirthDate;
+
+  return parentDates;
+}
+
+export function GetPersonDates(personName: string | null, parent: string) {
+  switch (parent) {
+    case 'mother':
+      return ExtractDates(personName);
+    case 'father':
+      return ExtractDates(personName);
```

Comment:
The `GetPersonDates` function has duplicated logic for 'mother' and 'father' cases. Since both cases call `ExtractDates` with the same argument, you can simplify the switch statement by removing the cases and directly returning `ExtractDates(personName)`.